content/blog/2020-10-26-october-20-community-gems.md

```diff
@@ -0,0 +1,163 @@
+artifacts in your project. These `.dvc` files contain pointers to your artifacts
+in remote storage (we use a simple content-based storage scheme). Because we use
+content-based storage, the remote storage itself isn't easy to browse (although
+[there are some discussions](https://github.com/iterative/dvc/issues/3621) about
+how to make the content more "discoverable").
+
+Your `.dvc` files are needed to establish meaningful links between
+human-readable filenames and file contents in remote storage, as well as to use
+Git versioning on your stored datasets and models. You can think of your DVC
+remote storage as a _compliment_ to your Git repository, not a replacement.
+
+In other words... if you're not Git versioning your `.dvc` files, you're not
+versioning anything in DVC remote storage!
+
+### [Q: Can I limit the number of network connections used by DVC during `dvc pull`?](https://discordapp.com/channels/485586884165107732/485596304961962003/739760523293360182)
+
+Yep- by default, DVC data transfer operations use a number of threads
+proportional to the number of CPUs detected. But, there's a handy flag for
+`dvc pull` and `dvc push` that lets you override the defaults:
+
+```dvc
+-j <number>, --jobs <number> - number of threads to run simultaneously to handle the downloading of files from the remote. The default value is 4 * cpu_count(). For SSH remotes, the default is just 4. Using more jobs may improve the total download speed if a combination of small and large files are being fetched.
+
+```
+
+### [Q: I'm working on a multi-class classification task. Can `dvc plots` show multiple precision recall curves- one for each class?](https://discordapp.com/channels/485586884165107732/485596304961962003/765117500530491472)
+
+Currently, `dvc plots` doesn't support multiple linear curves on a single plot
+(except for `dvc plots diff`, of course!). But, you could make one precision
+recall curve per class and display them side-by-side.
+
+To do this, you'd want to write the precision recall curve values to separate
+files for each class (`prc-0.json`,`prc-1.json`, etc.). Then you would run:
+
+```dvc
+$ dvc plots show 0.json 1.json
+```
+
+And you'll see two plots side-by-side! A benefit of this approach is that when
+you run `dvc plots diff` to compare precision recall curves across Git commits,
+you'll get a comparison plotted for each class.
+
+### [Q: Are you sure I should commit my `.dvc/config` file? It contains my logging credentials for storage, and I'm nervous about adding it to a shared Git repository.](https://discordapp.com/channels/485586884165107732/563406153334128681/768770079596740650)
+
+This is a common scenario- you don't necessarily want to broadcast your remote
+storage credentials to everyone on your team, but you still want to check-in
+your DVC setup (meaning, your `.dvc/config` file). In this case, you want to use
+a `local` config file!
+
+You can use the command
```

Comment:
we might want to mention about the alternatives `--system` and `--global`

Reply:
good idea